pagopa/io-app
# fix: [#177161429] Fixed missing loading on delete credit card

## Short description
Show loading when canceling a credit card.

**After clicking this delete button**
<img width="300" alt="Screenshot 2021-04-13 at 14 50 33" src="https://user-images.githubusercontent.com/61834253/114555226-c83df400-9c67-11eb-8123-cdf9df86e27a.png">

**A loading page like this will be shown**
<img width="300" alt="Screenshot 2021-04-13 at 14 56 01" src="https://user-images.githubusercontent.com/61834253/114555902-721d8080-9c68-11eb-8815-f01dfa7b53c8.png">

## How to test
Move into credit card detail and try to delete it. I suggest you to add a delay in dev server to slow down the deletion process and be able to properly observe the loading behavior
